pkg/lwip/Makefile

```diff
@@ -3,9 +3,33 @@ PKG_URL=git://git.savannah.nongnu.org/lwip.git
 PKG_VERSION=STABLE-2_0_3_RELEASE
 PKG_LICENSE=BSD-3-Clause
 
-.PHONY: all
+LWIP_SUBMODULES := $(patsubst makefiles/%.mk,%,$(wildcard makefiles/lwip_*.mk))
+LWIP_USEMODULE := $(filter $(LWIP_SUBMODULES),$(USEMODULE))
 
-all: git-download
-	"$(MAKE)" -C $(PKG_BUILDDIR)
+.PHONY: all $(LWIP_SUBMODULES)
+
+all: git-download $(LWIP_USEMODULE)
+	"$(MAKE)" -f $(CURDIR)/makefiles/Makefile.lwip $ -C $(PKG_BUILDDIR)
```

Comment:
I think this `$` in the middle is a copy paste leftover.

Reply:
I tested with Ubuntu 17.10 (`gcc (Ubuntu 7.2.0-8ubuntu3.2) 7.2.0`) and Ubuntu 16.04 (`gcc (Ubuntu 5.4.0-6ubuntu1~16.04.9) 5.4.0 20160609`.